apps/server/.env.test

```
@@ -19,4 +19,10 @@ SUPABASE_AUTH_EXTERNAL_GITHUB_CLIENT_ID="github-client"
 # Google OAuth
 # Google OAuth
 SUPABASE_AUTH_EXTERNAL_GOOGLE_CLIENT_ID="google-client"
-SUPABASE_AUTH_EXTERNAL_GOOGLE_SECRET="google-secret"
\ No newline at end of file
+SUPABASE_AUTH_EXTERNAL_GOOGLE_SECRET="google-secret"
+
+# API Games
+API_GAMES_MERCHANT="merchant"
+API_GAMES_SECRET="secret"
```

```suggestion
API_GAMES_SECRET="secret"
```suggestion
API_GAMES_SECRET="secret"
```

```